src/main/java/duke/command/dishesCommand/AddDishCommand.java

```diff
@@ -0,0 +1,54 @@
+package duke.command.dishesCommand;
+
+import duke.Dishes.DishList;
+import duke.Dishes.Dishes;
+import duke.command.Cmd;
+import duke.exception.DukeException;
+import duke.storage.Storage;
+import duke.ui.Ui;
+
+public class AddDishCommand extends Cmd<DishList> {
+
+    private Dishes dish;
+    private int amount;
+    private int Nb;
+
```

Comment:
What is `Nb`? Better to use a more descriptive name. That way even someone who's reading your code for the first time can understand what's going on in that part of the code